_dev/src/store/modules/product-feed/actions.ts

```diff
@@ -130,18 +130,29 @@ export default {
     const productFeedSettings = state.settings;
     const targetCountries = changeCountriesNamesToCodes(getters.GET_TARGET_COUNTRIES);
     const attributeMapping = getDataFromLocalStorage('productFeed-attributeMapping') || {};
+    const getShippingSettings = productFeedSettings.shippingSettings;
+    const deliveryFiltered = productFeedSettings.deliveryDetails.filter(
+      (e) => e.enabledCarrier
+      && ((e.minHandlingTimeInDays !== null && e.maxHandlingTimeInDays === null)
+      || (e.minHandlingTimeInDays !== null && e.maxHandlingTimeInDays !== null)),
```

Comment:
Can you double check this test? The same variable are used (only handling times) and the only one of them has `===`

Reply:
I will change it now